src/manager/checkpoint.rs

```diff
@@ -72,8 +72,20 @@ impl IntoSubsystem<anyhow::Error> for CheckpointSubsystem {
             log::debug!("We have {} subnets to manage", manage_subnet_futures.len());
 
             // Spawn a task to drive the `manage_subnet` futures.
-            let manage_subnets_task =
-                tokio::spawn(manage_subnet_futures.collect::<Vec<Result<()>>>());
+            let manage_subnets_task = tokio::spawn(async move {
+                loop {
+                    match manage_subnet_futures.next().await {
+                        Some(Ok(())) => {}
+                        Some(Err(e)) => {
+                            log::error!("Error in manage_subnet: {}", e);
```

Comment:
This great. Is it possible to tag from which ` manage_subnet` instance is the error?